.travis.yml

```diff
@@ -33,8 +33,7 @@ addons:
 script: ./travis.sh 
 
 notifications:
-  email:
-    - cukes-devs@googlegroups.com
+  email: false
```

Comment:
I share the desire for both CI systems to behave the same there, but I cannot see how to accomplish that from the AppVeyor docs. So I'd rather have at least one of them (Travis) be as useful as possible instead of both be equally uncommunicative (i.e. behaving no better than the lowest common demoninator IMO translates to "perfect is the enemy of good").

Maybe it is possible with AppVeyor, but it is not (easily) findable in the documentation.